server/service/apple_mdm.go

```diff
@@ -431,7 +431,7 @@ func (svc *Service) NewMDMAppleConfigProfile(ctx context.Context, teamID uint, r
 			msg := SameProfileNameUploadErrorMsg
 			if re, ok := existsErr.(interface{ Resource() string }); ok {
 				if re.Resource() == "MDMAppleConfigProfile.PayloadIdentifier" {
-					msg = "Couldn't upload. A configuration profile with this identifier (PayloadIdentifier) already exists."
+					msg = "Couldn't add. A configuration profile with this name already exists (PayloadDisplayName for .mobileconfig and file name for .json and .xml)."
```

Comment:
Maybe that was already discussed and we still decided to go with that message, but just in case - this error here is really because the `PayloadIdentifier` was a duplicate, not the name:

https://github.com/fleetdm/fleet/blob/b294ac97a9fa9e1a2fec87ebdf935517d4039bd0/server/datastore/mysql/apple_mdm.go#L124-L129

Something that can only happen with Apple profiles, so the error is less precise, but it's even misleading as the `name` might not even be a duplicate.

Reply:
ah you're right. I actually dont think we need this change. thanks for catching